src/main/java/org/apache/maven/shared/jarsigner/JarSignerUtil.java

```diff
@@ -200,37 +174,32 @@ public static boolean isArchiveSigned( final File jarFile )
      * @param entryName The name of the JAR file entry to check, must not be <code>null</code>.
      * @return <code>true</code> if the entry is related to a signature, <code>false</code> otherwise.
      */
-    protected static boolean isSignatureFile( String entryName )
-    {
-        if ( entryName.regionMatches( true, 0, "META-INF", 0, 8 ) )
-        {
-            entryName = entryName.replace( '\\', '/' );
-
-            if ( entryName.indexOf( '/' ) == 8 && entryName.lastIndexOf( '/' ) == 8 )
-            {
-                return endsWithIgnoreCase( entryName, ".SF" ) || endsWithIgnoreCase( entryName, ".DSA" )
-                    || endsWithIgnoreCase( entryName, ".RSA" ) || endsWithIgnoreCase( entryName, ".EC" );
+    protected static boolean isSignatureFile(String entryName) {
+        if (entryName.regionMatches(true, 0, "META-INF", 0, 8)) {
+            entryName = entryName.replace('\\', '/');
```

Comment:
This replacement is invalid, it should be dropped.

Reply:
Please file an issue for that. These lines are changing only because parent pom 39 requires reformatting. Byte code should be identical.